src/reconstruction/enzyme_constrained.jl

```diff
@@ -6,15 +6,22 @@ given by GECKO algorithm (see [`EnzymeConstrainedModel`](@ref) documentation for
 details). Multiple capacity constraints can be placed on the model using the
 kwargs.
 
-# Arguments
-- a `model` that implements the accessors `gene_product_molar_mass`,
-  `reaction_isozymes`, `gene_product_lower_bound`, `gene_product_upper_bound`.
+# Mandatory arguments
+A `model` that implements the accessors `gene_product_molar_mass`,
+`reaction_isozymes`, `gene_product_lower_bound`, `gene_product_upper_bound`.
+# Optional arguments
+There are two acceptable variants, (see the description of the kwargs below):
+1. `gene_product_mass_group` and `gene_product_mass_group_bound` are assigned.
+2. `total_enzyme_capacity` is assigned.
+
+## Description of kwargs
```

Comment:
```suggestion
Parameters `gene_product_mass_group` and `gene_product_mass_group_bound` specify the groups of gene products and the respective total limit of total mass of the gene products for each group. Gene products not listed in any gene product mass group are ignored.

For simplicity in many use cases, specifying the `total_enzyme_capacity` argument overrides the above arguments by specifying a single group called `uncategorized` of all gene products, with the corresponding mass bound.
```

Btw it might be cool to unify the naming of "bound" and "capacity" here.